app/uk/gov/hmrc/merchandiseinbaggage/views/SearchGoodsCountryView.scala.html

```diff
@@ -57,9 +58,7 @@
 
         @errorSummary(form.errors, Some("country-auto-complete"))
 
-        @h1(messages("searchGoodsCountry.heading", goodsCategory))
-
-        <p class="govuk-hint">@messages("searchGoodsCountry.hint")</p>
```

Comment:
this hint still needs to stay for import so you can just wrap in an if(declarationType == Import)